props/saber_fett263_buttons.h

```diff
@@ -257,7 +325,100 @@ SaberFett263Buttons() : PropBase() {}
       case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
         previous_preset();
         return true;
+
+       // Gesture Controls
+		
+       // Battle Mode
+		    
+       #ifdef FETT263_BATTLE_MODE
```

Comment:
All preprocessor statements (like #ifdef) should be un-indented (start at the beginning of the line)